sql/moz-fx-data-marketing-prod/ga_derived/www_site_downloads_v2/metadata.yaml

```diff
@@ -8,6 +8,9 @@ labels:
   owner1: kwindau@mozilla.com
 scheduling:
   dag_name: bqetl_google_analytics_derived_ga4
+  depends_on:
+  - task_id: wait_for_prod_events_partition
+    dag_name: bqetl_google_analytics_derived_ga4
```

Comment:
I personally like having it this way because it's easy to read exactly how it's calculated directly from the source without having to go to an intermediate which kind of obfuscates what is happening, so I hope it's ok if I leave it like this for now!

Reply:
That's fine, since there doesn't appear to be much duplicated logic in this case.